base/replacer/replacer.Deployment.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    description: Backend for replace operations.
+  labels:
+    deploy: sourcegraph
+  name: replacer
+spec:
+  minReadySeconds: 10
+  replicas: 1
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      app: replacer
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+    type: replacer
+  template:
+    metadata:
+      labels:
+        app: replacer
+    spec:
+      containers:
+      - env:
+        - name: REPLACER_CACHE_SIZE_MB
+          value: "100000"
+        - name: POD_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
+        - name: CACHE_DIR
+          value: /mnt/cache/$(POD_NAME)
+        image: index.docker.io/sourcegraph/replacer:3.7.0@sha256:f193699e67...FIXME
+        terminationMessagePolicy: FallbackToLogsOnError
+        name: searcher
+        ports:
+        - containerPort: 3185
+          name: http
+        - containerPort: 6060
+          name: debug
+        readinessProbe:
+          failureThreshold: 1
+          httpGet:
+            path: /healthz
+            port: http
+            scheme: HTTP
+          periodSeconds: 1
+        resources:
+          limits:
+            cpu: "4"
+            memory: 500M
```

Comment:
Replacer is CPU heavy so I'm choosing 4 for cpu (cf. 2 for searcher). Reasonable? Needs very little memory, so choosing 500M (cf. 1G for searcher).